File: java_project/src/main/java/com/example/MainSample.java
package com.example;

import com.example.grade.GradeReportSample;

public class MainSample {

    @SuppressWarnings("CallToPrintStackTrace")
    public static void main(String[] args) {
        try {
            System.out.println("=== 成績表プログラム 開始 ===");

            // インプット情報を定義
            String className = "数学";
            int totalClassCount = 5;
            String studentName = "山田太郎";
            int examScore = 80;
            int attendanceCount = 5;

            // 成績判定を呼び出し
            GradeReportSample gradeReports = new GradeReportSample(className, totalClassCount);
            gradeReports.judgeGrade(studentName, examScore, attendanceCount);
            gradeReports.judgeGrade(studentName, examScore, attendanceCount);
            

            // 順位表の中身をコンソールに出力
            gradeReports.showRanking();

            System.out.println("=== 成績表プログラム 終了 ===");            
        } catch (Exception e) {
            e.printStackTrace();
        }
    }

}

File: python_project/src/main_sample.py
from grade.grade_report_sample import GradeReportSample

def main():
    try:
        print("=== 成績表プログラム 開始 ===")
        
        # 入力値を定義
        class_name = "数学"
        total_class_count = 5
        student_name = "山田太郎"
        exam_score = 80
        attendance_count = 5

        # 成績判定を呼び出し
        grade_report_sample = GradeReportSample(class_name, total_class_count)
        grade_report_sample.judge_grade(student_name, exam_score, attendance_count)
        grade_report_sample.judge_grade(student_name, exam_score, attendance_count)

        # 順位表を表示
        grade_report_sample.show_ranking()

        print("=== 成績表プログラム 終了 ===")
    except Exception as e:
        e.print_stack()


if __name__ == "__main__":
    main()
